src/theme/theme.ts

```diff
@@ -69,7 +69,8 @@ const theme: ITheme = {
       red: '#EB5757',
       orange: '#f2994a',
       purple: '#9B51E0',
-      yellow: '#FFF066',
+      yellow: '#FFE600',
```

Comment:
this yellow was just wrong, see [Figma](https://www.figma.com/file/mvY1f3xMXUEd6szvT82FxB/Hopsworks-UI?node-id=9288%3A804)